src/Classes/RichTextViewController.swift

```diff
@@ -218,7 +230,7 @@ public class RichTextViewController: UIViewController {
     /// - parameter selection: An `NSRange` to check
     ///
     /// - returns: True if selection contains at least 1 numbered list, false otherwise
-    private func selectionContainsNumberedList(var selection: NSRange) -> Bool {
+    public func selectionContainsNumberedList(var selection: NSRange) -> Bool {
```

Comment:
I this case, I wouldn't make the input a `var` so you can change it, just assign it to a local var in the func